server/monitor.go

```diff
@@ -2095,18 +2101,19 @@ type LeafzOptions struct {
 
 // LeafInfo has detailed information on each remote leafnode connection.
 type LeafInfo struct {
-	Name     string   `json:"name"`
-	IsSpoke  bool     `json:"is_spoke"`
-	Account  string   `json:"account"`
-	IP       string   `json:"ip"`
-	Port     int      `json:"port"`
-	RTT      string   `json:"rtt,omitempty"`
-	InMsgs   int64    `json:"in_msgs"`
-	OutMsgs  int64    `json:"out_msgs"`
-	InBytes  int64    `json:"in_bytes"`
-	OutBytes int64    `json:"out_bytes"`
-	NumSubs  uint32   `json:"subscriptions"`
-	Subs     []string `json:"subscriptions_list,omitempty"`
+	Name      string   `json:"name"`
+	IsSpoke   bool     `json:"is_spoke"`
+	Account   string   `json:"account"`
+	IP        string   `json:"ip"`
+	Port      int      `json:"port"`
+	RTT       string   `json:"rtt,omitempty"`
+	InMsgs    int64    `json:"in_msgs"`
+	OutMsgs   int64    `json:"out_msgs"`
+	InBytes   int64    `json:"in_bytes"`
+	OutBytes  int64    `json:"out_bytes"`
+	SentBytes int64    `json:"sent_bytes"`
```

Comment:
We doing leaf compression in this PR too?

Reply:
No, but since the count is on a `*client` it would be available to all type of connections, that's what I added it to all. And again, this value is simply the number of bytes wrote to the TCP connection, regardless if compression is used or not. It is different from OutBytes in that it contains ANY data written, so for messages that includes the MSG protocol and any overhead, etc..

Reply:
That being said, happy to remove from all objects except RouteInfo for the moment...

Reply:
ok so without compression you still think this could be valuable?

Reply:
I think remove for now from this PR and give some more thought.

Reply:
ok, will do.